Src/BootCamp.Chapter/TextBoxPrinter.cs

```diff
@@ -1,12 +1,30 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace BootCamp.Chapter
+﻿namespace BootCamp.Chapter
 {
     public static class TextBoxPrinter
```

Comment:
I actually like the way you split it a lot. Having a break definitely helped you a lot 😄

Reply:
Thanks